### PR TITLE
ufal/shibboleth-redirect-from-login

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  2  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |

### CODE REVIEW

File: src/aai/aai.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
     this.setup = function(options) {
       var opts = jQuery.extend({}, this.defaults, options),
           defaultCallback = function(e) {
-            window.location = opts.host + '/Shibboleth.sso/Login?SAMLDS=1&target=' + opts.target + '&entityID=' + window.encodeURIComponent(e.entityID);
+            window.location = opts.host + '/Shibboleth.sso/Login?SAMLDS=1&target=' + opts.target + '?redirectUrl=' + window.location.href + '&entityID=' + window.encodeURIComponent(e.entityID);
```
Comment (vidiecan) [inline src/aai/aai.js]: does not make sense

**Review by vidiecan — CHANGES_REQUESTED** [review]

File: src/aai/aai.js (hunk 1 above)
Comment (milanmajchrak) [reply src/aai/aai.js]: But it fixed problem

Comment (vidiecan) [inline src/aai/aai.js]: change it so it is more readable
target_url = opts.target + ?redirect...;
window.location = .... + target_url ?+ ....

**Review by vidiecan — CHANGES_REQUESTED** [review]

File: src/aai/aai.js (hunk 1 above)
Comment (milanmajchrak) [reply src/aai/aai.js]: Done

**Review by vidiecan — APPROVED** [review]

**Pull request merged**